### PR TITLE
Dx fixing direct credit card feedback

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: payments/card/redirect.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ sidebar:
       title: Seamless View
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
+      title: Capture
```
Comment (payex-dev) [inline payments/card/redirect.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 14
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497&bid=17141166)

File: payments/card/other-features.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ sidebar:
       title: Redirect
     - url: /payments/card/seamless-view
       title: Seamless View
-    - url: /payments/card/capture
-      title: Capture
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
```
Comment (payex-dev) [inline payments/card/other-features.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/after-payment.md [hunk 3]
```diff
@@ -10,10 +10,10 @@ sidebar:
       title: Redirect
     - url: /payments/card/seamless-view
       title: Seamless View
-    - url: /payments/card/capture
-      title: Capture
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
+      title: Capture
```
Comment (payex-dev) [inline payments/card/after-payment.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/capture.md [hunk 4]
```diff
@@ -10,10 +10,10 @@ sidebar:
       title: Redirect
     - url: /payments/card/seamless-view
       title: Seamless View
-    - url: /payments/card/capture
-      title: Capture
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
+      title: Capture
```
Comment (payex-dev) [inline payments/card/capture.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/index.md [hunk 5]
```diff
@@ -10,10 +10,10 @@ sidebar:
       title: Redirect
     - url: /payments/card/seamless-view
       title: Seamless View
-    - url: /payments/card/capture
-      title: Capture
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
```
Comment (payex-dev) [inline payments/card/index.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/mobile-card-payments.md [hunk 6]
```diff
@@ -12,6 +12,8 @@ sidebar:
       title: Seamless View
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
+      title: Capture
```
Comment (payex-dev) [inline payments/card/mobile-card-payments.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/after-payment.md [hunk 7]
```diff
@@ -10,10 +10,10 @@ sidebar:
       title: Redirect
     - url: /payments/card/seamless-view
       title: Seamless View
-    - url: /payments/card/capture
-      title: Capture
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
```
Comment (payex-dev) [inline payments/card/after-payment.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/capture.md [hunk 8]
```diff
@@ -10,10 +10,10 @@ sidebar:
       title: Redirect
     - url: /payments/card/seamless-view
       title: Seamless View
-    - url: /payments/card/capture
-      title: Capture
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
```
Comment (payex-dev) [inline payments/card/capture.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/redirect.md [hunk 9]
```diff
@@ -12,6 +12,8 @@ sidebar:
       title: Seamless View
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
```
Comment (payex-dev) [inline payments/card/redirect.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/seamless-view.md [hunk 10]
```diff
@@ -10,10 +10,10 @@ sidebar:
       title: Redirect
     - url: /payments/card/seamless-view
       title: Seamless View
-    - url: /payments/card/capture
-      title: Capture
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
+      title: Capture
```
Comment (payex-dev) [inline payments/card/seamless-view.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/index.md [hunk 11]
```diff
@@ -10,10 +10,10 @@ sidebar:
...
+      title: Capture
```
Comment (payex-dev) [inline payments/card/index.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/seamless-view.md [hunk 12]
```diff
@@ -10,10 +10,10 @@ sidebar:
       title: Redirect
     - url: /payments/card/seamless-view
       title: Seamless View
-    - url: /payments/card/capture
-      title: Capture
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
```
Comment (payex-dev) [inline payments/card/seamless-view.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/other-features.md [hunk 13]
```diff
@@ -10,10 +10,10 @@ sidebar:
...
+      title: Capture
```
Comment (payex-dev) [inline payments/card/other-features.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/mobile-card-payments.md [hunk 14]
```diff
@@ -12,6 +12,8 @@ sidebar:
       title: Seamless View
     - url: /payments/card/direct
       title: Direct
+    - url: /payments/card/capture
```
Comment (payex-dev) [inline payments/card/mobile-card-payments.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/direct.md [hunk 15]
```diff
@@ -119,23 +122,23 @@ Content-Type: application/json
...
-            "description": "books & ink",
-            "payeeReference": "cyrusLibrary1583846025",
+            "description": "Test transaction",
+            "payeeReference": "1583846025",
             "isOperational": false,
             "operations": [
                 {
                     "method": "PATCH",
-                    "href": "https://api.stage.payex.com/psp/creditcard/payments/45635233-1aa8-46a2-9584-08d7c4e16b79/authorizations/84c440a1-5745-4a88-f4ed-08d7c4dcbf36",
+                    "href": "https://api.stage.payex.com/psp/creditcard/payments/{{ page.payment_id }}/authorizations/84c440a1-5745-4a88-f4ed-08d7c4dcbf36",
```
Comment (asbjornu) [inline payments/card/direct.md]:
```suggestion
                    "href": "https://api.stage.payex.com/psp/creditcard/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
```

File: payments/card/direct.md [hunk 16]
```diff
@@ -144,14 +147,18 @@ Content-Type: application/json
 }
 ```
 
-If there is 3-D secure authentication, the request and response headers
-will be as shown in the examples below. Notice that here the `redirect-authentication`
+## Step 2b: Create an authorization transaction with 3-D secure
+
+If 3-D secure authentication is needed, the `rel` is set to
+`redirect-authentication` and the `state` is `AwaitingActivity`. This means that
+you will be redirected to complete the 3-D secure authentication. See the
```
Comment (asbjornu) [inline payments/card/direct.md]:
```suggestion
the payer will have to be redirected to complete the 3-D secure authentication. See the
```

File: payments/card/direct.md [hunk 17]
```diff
@@ -198,23 +205,23 @@ Content-Type: application/json
         "panToken": "cca2d98d-8bb3-4bd6-9cf3-365acbbaff96",
         "panEnrolled": true,
         "acquirerTransactionTime": "0001-01-01T00:00:00Z",
-        "id": ""/psp/{{ instrument }}/payments/{{ page.payment_id }}/{{ page.transaction_id }}",
+        "id": ""/psp/creditcard/payments/{{ page.payment_id }}/{{ page.transaction_id }}",
```
Comment (asbjornu) [inline payments/card/direct.md]:
```suggestion
        "id": ""/psp/creditcard/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
```

File: payments/card/direct.md [hunk 18]
```diff
@@ -198,23 +205,23 @@ Content-Type: application/json
...
         "transaction": {
-            "id": ""/psp/{{ instrument }}/payments/{{ page.payment_id }}/{{ page.transaction_id }}",
+            "id": ""/psp/creditcard/payments/{{ page.payment_id }}/{{ page.transaction_id }}",
```
Comment (asbjornu) [inline payments/card/direct.md]:
```suggestion
            "id": ""/psp/creditcard/payments/{{ page.payment_id }}/transactions/{{ page.transaction_id }}",
```

File: payments/card/direct.md [hunk 19]
```diff
@@ -198,23 +205,23 @@ Content-Type: application/json
...
-            "description": "books & ink",
-            "payeeReference": "cyrusLibrary1583846100",
+            "description": "Test transaction",
+            "payeeReference": "1583846100",
             "isOperational": true,
             "operations": [
                 {
                     "method": "GET",
-                    "href": "https://api.stage.payex.com/psp/{{ payment_instrument }}/confined/payments/authorizations/authenticate/{{ page.payment_id }}",
+                    "href": "https://api.stage.payex.com/psp/creditcard/confined/payments/authorizations/authenticate/{{ page.payment_id }}",
```
Comment (asbjornu) [inline payments/card/direct.md]:
```suggestion
                    "href": "https://api.stage.payex.com/psp/creditcard/confined/payments/authorizations/authenticate/{{ page.transaction_id }}",
```

File: payments/card/direct.md [hunk 20]
```diff
@@ -144,14 +147,18 @@ Content-Type: application/json
 }
 ```
 
-If there is 3-D secure authentication, the request and response headers
-will be as shown in the examples below. Notice that here the `redirect-authentication`
+## Step 2b: Create an authorization transaction with 3-D secure
+
+If 3-D secure authentication is needed, the `rel` is set to
```
Comment (asbjornu) [inline payments/card/direct.md]:
```suggestion
If 3-D Secure authentication is needed, the `rel` is set to
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 14
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497&bid=17141166)

File: payments/card/index.md (hunk 11 above)
Comment (payex-dev) [inline payments/card/index.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/after-payment.md (hunk 3 above)
Comment (payex-dev) [inline payments/card/after-payment.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/seamless-view.md (hunk 10 above)
Comment (payex-dev) [inline payments/card/seamless-view.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/index.md (hunk 5 above)
Comment (payex-dev) [inline payments/card/index.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/redirect.md (hunk 9 above)
Comment (payex-dev) [inline payments/card/redirect.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/capture.md (hunk 4 above)
Comment (payex-dev) [inline payments/card/capture.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/mobile-card-payments.md (hunk 14 above)
Comment (payex-dev) [inline payments/card/mobile-card-payments.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/other-features.md (hunk 2 above)
Comment (payex-dev) [inline payments/card/other-features.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/redirect.md (hunk 1 above)
Comment (payex-dev) [inline payments/card/redirect.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/other-features.md (hunk 13 above)
Comment (payex-dev) [inline payments/card/other-features.md]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/capture.md (hunk 8 above)
Comment (payex-dev) [inline payments/card/capture.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/after-payment.md (hunk 7 above)
Comment (payex-dev) [inline payments/card/after-payment.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

File: payments/card/seamless-view.md (hunk 12 above)
Comment (payex-dev) [inline payments/card/seamless-view.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5228497)

**Pull request merged**